d1: add a changelog (#9230)
cloudflare/cloudflare-docs: content/d1/changelog.md

---
pcx_content_type: changelog
title: Changelog
weight: 11
rss: file
---

# Changelog

## 2023-05-19

### New experimental backend

D1 has a new experimental storage backend that dramatically improves query throughput, latency and reliability. The experimental backend will become the default backend in the near future. To create a database using the experimental backend, use `wrangler` and set the `--experimental-backend` flag when creating a database: 

```sh
$ wrangler d1 create your-database --experimental-backend
```

> You can read more about the experimental backend in the [announcement blog](https://blog.cloudflare.com/d1-turning-it-up-to-11/).

Reviewer: ```suggestion
Read more about the experimental back end in the [announcement blog](https://blog.cloudflare.com/d1-turning-it-up-to-11/).
```
